Frontend/user/src/components/Category/BaseCategory.vue

```diff
@@ -132,10 +132,7 @@ onMounted(() => {
 });
 
 const idDefine = (index: number, idx: number | undefined = undefined) => {
-  if (idx !== undefined) {
-    return `id-${index}-${idx}`;
-  }
-  return `id-${index}`;
+  return !idx ? `id-${index}-${idx}` : `id-${index}`;
```

Comment:
It seems this logic is different from the old code

Reply:
This logic was merged at the meeting and demo example with Minh and this was merged into the dev branch by Minh
When I checkout to the "refactor/products/tags" branch from the dev branch, it was there before

Reply:
I see the logic of these two functions are the same